Libraries/Connect/PackageInternal/Envelope.swift

```diff
@@ -85,6 +85,26 @@ public enum Envelope {
         }
     }
 
+    /// Determines whether the specified data contains more than 1 message.
+    ///
+    /// - parameter packedData: The packed data to analyze.
+    ///
+    /// - returns: True if the data contains more than 1 message.
+    public static func containsMultipleMessages(_ packedData: Data) -> Bool {
```

Comment:
These are public so the `ConnectGRPC` module can see them. Everything in this directory can be switched to using `package` visibility once it's widely supported

Reply:
I'm confused by this function. It looks like it would always return true for gRPC-Web and Connect protocols, since those encode the trailers/status as a second (ostensibly final) message in the stream? Or is there some other layer that sees the "end stream" flag in the next envelope and truncates the data that this helper would see?

Reply:
Updated